formatTest/unit_tests/expected_output/wrappingTest.re

```diff
@@ -252,62 +252,56 @@ let defOptionalAliasAnnot =
   10;
 
 /* Invoking them */
-named
+named(
```

Comment:
Definitely improves this case too. Now the comment on the first arg is in the correct location when printed.